_includes/filter-controls.html

```diff
@@ -0,0 +1,94 @@
+<div id="filter-controls">
```

Comment:
Should filters be a part of the main content? Seems kinda annoying that I need to tab through all the filters first before I can reach the guidelines if I am using a keyboard, or the filters could have something like "Skip to guidelines" link

Reply:
People can use the Page Content navigation for that. Do you think an extra skip link is needed?

Reply:
I meant if you do "Skip to main content", you would either need to tab through all filters or tab back through multiple links to the "Page Contents", another skip to link would simplify things. I am not insisting on this change,  it can be a later improvement if someone else complains :)